hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiryRecordsHelper.java

```diff
@@ -105,7 +102,7 @@ public void streamCryptoRenewal(
                         ? syntheticTxnFactory.synthContractAutoRenew(entityNum, newExpiry, payerId)
                         : syntheticTxnFactory.synthAccountAutoRenew(entityNum, newExpiry);
         final var expirableTxnRecord =
-                forTouchedAccount(grpcId, eventTime)
+                forTouchedAccount(grpcId, eventTime, payerForAutoRenew.toEntityId())
```

Comment:
nit : since everyplace of `AutoRenew` is changed to `expiry` can change `payerForAutoRenew` to `payerForExpiry`?